Library/Formula/cmake.rb

```diff
@@ -34,7 +34,10 @@ class Cmake < Formula
   option "without-docs", "Don't build man pages"
   depends_on :python => :build if MacOS.version <= :snow_leopard && build.with?("docs")
   depends_on "xz" # For LZMA
-  depends_on "qt" => :optional
+
+  # The GUI option previously offered with `--with-qt` is no longer offered due to
+  # potential future failure and misery. For the GUI option, please tap `caskroom/cask`
+  # and do `brew cask install cmake`. Thanks!
```

Comment:
I thought ` failure and misery ` sounded more dooming, but sure, we can roll with your suggestion :wink: